language_bindings/rust/rust_runtime/src/runtime_bindings.rs

```diff
@@ -0,0 +1,300 @@
+
+//! runtime_bindings is a manually generated using bindgen on halide_runtime.h.
+//!
+//! Eventually automatically generating this file may be a better approach, but may introduce stability issues.
+    #![allow(dead_code)]
+
+    #![allow(non_camel_case_types)]
+    #![allow(non_upper_case_globals)]
+
+    extern "C" {
+        pub fn halide_malloc(
+            user_context: *mut ::std::os::raw::c_void,
+            x: isize,
+        ) -> *mut ::std::os::raw::c_void;
+    }
```

Comment:
If this definition is provided by libhalide.a, then this crate must have a build script which instructs cargo to link libhalide.a.